docs/source/examples/BSTLD.rst

```diff
@@ -13,118 +13,118 @@ images. (:numref:`Fig. %s <example-Box2D>`)
 
    The preview of an image with Box2D annotations.
 
-*************
- Preparation
-*************
+Here, we take ``BSTLD`` as an example to show how you can operate on
+a dataset with :ref:`reference/label_format:Box2D` label.
+See `this page <https://www.graviti.cn/open-datasets/BSTLD>`_  for more details about this dataset.
 
-First of all, create a :ref:`features/tensorbay_client:GAS Client`.
-
-.. code:: python
-
-    >>> from graviti import GAS
-
-    >>> ACCESS_KEY = "Accesskey-*****"
-    >>> gas = GAS(ACCESS_KEY)
+**************************
+ Authorize a client object
+**************************
 
-*****************************
- Read Dataset from TensorBay
-*****************************
+First of all, create a :ref:`features/tensorbay_client:GAS Client`.
 
-As mentioned in :ref:`quick_start:Quick Start`, obtain_ and fork_ should be done before reading a
-dataset from TensorBay. Then, pass the correct dataset name to the GAS client, and you will get a
-:ref:`features/tensorbay_client:Dataset Client`.
+.. literalinclude:: ../../../examples/bstld.py
+   :language: python
+   :lines: 8, 12-13
 
-Here, we take `BSTLD`_ as an example.
+*****************
+ Create a Dataset
+*****************
 
-.. _fork: https://docs.graviti.cn/guide/opendataset/fork
+.. literalinclude:: ../../../examples/bstld.py
+   :language: python
+   :lines: 15
 
-.. _BSTLD: https://www.graviti.cn/open-datasets/BSTLD
+*******************
+ List Dataset names
+*******************
 
-.. _obtain: https://docs.graviti.cn/guide/opendataset/get
+You can use the list method to print all your forked open datasets.
 
-.. code:: python
+.. literalinclude:: ../../../examples/bstld.py
+   :language: python
+   :lines: 17
 
-   >>> dataset_client = gas.get_dataset("BSTLD")
+.. note::
 
-If you are not sure about the dataset name, you can visit our `Opendataset Platform`_ to check all
-"forkable" open datasets.
+   Note that Method ``list_dataset_names()`` returns an iterator, use ``list()`` to transfer it to a "list".
 
-.. _opendataset platform: https://www.graviti.cn/open-datasets
+**********************
+Organize Local Dataset
+**********************
 
-You can use the list method to print all your forked open datasets.
+If there is no :ref:`reference/glossary:Dataloader` available to your target dataset,
```

Comment:
Instead of the label type, the theme of this chapter should be the dataset.
The following description is better.

Before uploading the BSTLD dataset to TensorBay, we first need to organize it using the ``Dataset`` class.
It takes the following steps to organize BSTLD.